Cloudinary/Integration/02_ManageCloudinaryAttributes.md

```diff
@@ -0,0 +1,113 @@
+
+  > [Info] Be aware that only attributes which are neither multi-language nor multi-channel can be used as defining attributes for variants. For detailed information, see [Create a variant set](./07_ManageVariantSets.md#create-a-variant-set).
+
+6. Enable the *Active* toggle to switch the attribute active after creation.
+
+7. Click the ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned sets* field. The button is locked if you have not yet selected a data type.   
+  A drop-down list with all active attribute sets is displayed.
+
+8. Select an attribute set in the *Assigned sets* drop-down list.
+
+  > [Info] You can assign the attribute to multiple sets. Repeat the steps **7** to **8** to assign the attribute to a further attribute set. To delete the assignment to a selected set, click the ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete) button right to the set.
+
+9. Configure the appropriate settings in the *CONFIGURATION* section. The configuration can also be edited subsequently.       
+For detailed information about the Cloudinary configuration settings, see [Configure a Cloudinary attribute](#configure-a-cloudinary-attribute).
+
+10. Click the [SAVE] button in the upper right corner.   
+  The new Cloudinary attribute has been saved. The *Create attribute* view is closed.  
+
+11. Press **F5** to initialize the *Core1 Platform* and to apply the changes.   
+
+
+## Configure a Cloudinary attribute
+
+The configuration of the Cloudinary attribute is important to define the meta data fields and other exchange settings between Actindo and Cloudinary.
+
+#### Prerequisites
+
+A Cloudinary attribute has been created, see [Create a Cloudinary attribute](#create-a-cloudinary-attribute), or at least the steps **1** to **8** of the [Create a Cloudinary attribute](#create-a-cloudinary-attribute) procedure have been executed.
+
+#### Procedure
+
+*PIM > Settings > Tab ATTRIBUTES > Button Add*
+*PIM > Settings > Tab ATTRIBUTES > Select Cloudinary attribute*
+
+![Cloudinary configuration](../../Assets/Screenshots/PIM/Settings/Attributes/CloudinaryConfiguration.png "[Cloudinary configuration]")
+
+> [Info] The following configuration settings are identical for the *Cloudinary Image* and the *Cloudinary Video* data type.
+
+1. Navigate to the *CONFIGURATION* section of the Cloudinary attribute.
+
+2. Enter a folder name where the Cloudinary files are put into in the *Folder* field. If the folder is not yet created in Cloudinary, it is automatically created when uploading the first file. Use placeholders to define product specific folder, for instance **{sku}**.
+
+3. Enable the *Read-only* toggle to allow no manual modification, for instance if an automatic import from Cloudinary is configured and the imported files should not be editable.
```

Comment:
```suggestion
3. Enable the *Read-only* toggle to prevent any manual modification, for instance if an automatic import from Cloudinary is configured and the imported files should not be editable.
```